packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -300,20 +300,19 @@ class C4DMastheadComposite extends HostListenerMixin(LitElement) {
     const { heading } = megapanel;
     const { viewAll, highlights } = menu;
 
+    const { title } = this.getL0Data()[_parentKey] || {};
+
     return html`
       <c4d-megamenu layout="${MEGAMENU_LAYOUT_SCHEME.LIST}">
-        ${highlights
-          ? html`
-              <c4d-megamenu-left-navigation>
-                ${highlights.map((group, i) =>
-                  this._renderMegapanelLinkGroup(group, {
-                    headingLevel: 2,
-                    autoid: `${c4dPrefix}--masthead__l0-nav-list${i}`,
-                  })
-                )}
-              </c4d-megamenu-left-navigation>
-            `
-          : null}
+        <c4d-megamenu-left-navigation>
+          <span>${title}</span>
```

Comment:
Is this supposed to be a link? Also, have we considered if this should be a heading?

Reply:
It should not be a link, I considered a heading, but wasn't sure if that was the right semantics. Considering also adding `aria-hidden` to make this a presentation-only element.

Reply:
Right, on closer look that seems fine. Especially considering the `<c4d-megamenu-right-navigation>` can have an `<h2>` (see below). I mean, if design is saying that the repetition is useful for sighted users, it should follow that its useful for non-sighted users, so I'd avoid using `aria-hidden`.

Also we should probably add a `part` attribute to the `<span>`.